src/styles/mixins/_map-button-themes.scss

```diff
@@ -4,7 +4,13 @@
         "towns": $color-secondary-teal-shade-3,
         "islands": #1066D6,
```

Comment:
Can we add these hex codes to our color tokens list?

Reply:
Have checked with Kyle

Reply:
Are you going to create a separate ticket for this if yes?

Reply:
Yep, will do